src/main/java/com/cascadebot/cascadebot/commands/core/InviteCommand.java

```diff
@@ -16,10 +16,7 @@
 
     @Override
     public void onCommand(Member sender, CommandContext context) {
-        public static String getInvite() {
-            // return String.format("https://discordapp.com/oauth2/authorize?client_id=%s&scope=bot&permissions=%s",
-                    CascadeBot.INS.getSelfUser().getId(), Permission.ALL_GUILD_PERMISSIONS);
-        }
+    //    CascadeBot.getInvite();
```

Comment:
Just surround this in `context,reply()` so we know what's going on

Reply:
yeep